src/lib/markets/graph.ts

```diff
@@ -143,9 +149,19 @@ export async function fetchActivePositionPnl({
   let netDeposits = 0n
   let keeperFees = snapshot.checkpoint.settlementFee
   let positionFees = snapshot.checkpoint.tradeFee
+
+  const tradeFeeData = calcTradeFee({
+    positionDelta: magnitude, // TODO: check this
+    marketSnapshot,
+    isMaker: side === PositionSide.maker,
+    direction: side,
+  })
   const pendingPriceImpactFee = calcPriceImpactFromTradeFee({
-    totalTradeFee: snapshot.pre.checkpoint.tradeFee,
```

Comment:
I don't think we need this change, the tradeFee from the snapshot should be the correct value. But since we need the component parts in the price impact calculation I think i'll need to figure out how to get the correct values here...